frontend/js/src/user/playlists/components/ImportAppleMusicPlaylistModal.tsx

```diff
@@ -0,0 +1,208 @@
+      );
+      toast.success(
+        <ToastMsg
+          title="Successfully imported playlist from Apple Music"
+          message={
+            <>
+              Imported
+              <a href={newPlaylist.identifier}> {playlistName}</a>
+            </>
+          }
+        />,
+        { toastId: "create-playlist-success" }
+      );
+      setNewPlaylists([...newPlaylists, newPlaylist]);
+    } catch (error) {
+      toast.error(
+        <ToastMsg
+          title="Something went wrong"
+          message={<>We could not save your playlist: {error.toString()}</>}
+        />,
+        { toastId: "save-playlist-error" }
+      );
+    }
+    setPlaylistLoading(null);
+  };
+
+  return (
+    <div
+      className={`modal fade ${modal.visible ? "in" : ""}`}
+      id="ImportMusicServicePlaylistModal"
+      tabIndex={-1}
+      role="dialog"
+      aria-labelledby="ImportMusicServicePlaylistLabel"
+      data-backdrop="static"
+    >
+      <div className="modal-dialog" role="document">
+        <div className="modal-content">
+          <div className="modal-header">
+            <button
+              type="button"
+              className="close"
+              data-dismiss="modal"
+              aria-label="Close"
+              onClick={resolveAndClose}
+            >
+              <span aria-hidden="true">&times;</span>
+            </button>
+            <h4
+              className="modal-title"
+              id="ImportMusicServicePlaylistLabel"
+              style={{ textAlign: "center" }}
+            >
+              Import playlist from Apple Music
+            </h4>
+          </div>
+          <div className="modal-body">
+            <p className="text-muted">
+              Add one or more of your Apple Music playlists below:
+            </p>
+            <div
+              className="list-group"
+              style={{ maxHeight: "50vh", overflow: "scroll" }}
+            >
+              {playlists?.map((playlist: AppleMusicPlaylistObject) => (
+                <button
+                  type="button"
+                  key={playlist.id}
+                  className="list-group-item"
+                  style={{
+                    display: "flex",
+                    justifyContent: "space-between",
+                  }}
+                  disabled={!!playlistLoading}
+                  name={playlist.attributes.name}
+                  onClick={() =>
+                    importTracksToPlaylist(
+                      playlist.id,
+                      playlist.attributes.name
+                    )
+                  }
+                >
+                  <span>{playlist.attributes.name}</span>
+                </button>
+              ))}
+            </div>
+            {!!playlistLoading && (
+              <div>
+                <p>
+                  Loading playlist {playlistLoading}... It might take some time
+                </p>
+                <Loader isLoading={!!playlistLoading} />
```

Comment:
Not important at all, and not worth changing really, but for reference you could skip this conditional since you already have `!!playlistLoading` for conditional renderin at line 184